### PR TITLE
Move to new version of named property in script files

### PR DESCRIPTION
**Pull request by dmcavoy**
Went through all the script files and changed:

`named_level 'Example Level'`
to
`level 'Example Level', named: true`

### CODE REVIEW

File: dashboard/config/scripts/csp1-2018.script [hunk 1]
```diff
@@ -68,60 +68,60 @@ stage 'Unit 1 Chapter 1 Assessment', lockable: true, flex_category: 'csp1_1'
...
 assessment 'U2L05 Assessment 2_2018', progression: 'Check Your Understanding'
 assessment 'U2L3 Assessment_2018', progression: 'Check Your Understanding'
 
 stage 'The Need for DNS', flex_category: 'csp1_2'
-named_level 'CSP U1L12 SFLP'
-named_level 'Internet Simulator: DNS_2018', progression: 'Internet Simulator: DNS'
-named_level 'The Internet: IP Addresses & DNS_2018', progression: 'The Internet: IP Addresses & DNS'
+level 'CSP U1L12 SFLP'
+level 'Internet Simulator: DNS_2018', progression: 'Internet Simulator: DNS'
+level 'The Internet: IP Addresses & DNS_2018', progression: 'The Internet: IP Addresses & DNS'
```
Comment (dmcavoy) [inline dashboard/config/scripts/csp1-2018.script]: @dmcavoy these 3 levels need named

File: dashboard/config/scripts/csp2-2017.script [hunk 2]
```diff
@@ -30,20 +30,20 @@ stage 'Encoding B&W Images', flex_category: 'csp2_1'
...
+level 'Widget: Black and White Pixelation', progression: 'Pixelation Widget: Black and White Pixelation', named: true
 assessment 'U1L14 Assessment 1', progression: 'Check Your Understanding'
 assessment 'U1L14 - Assessment 2', progression: 'Check Your Understanding'
 
 stage 'Encoding Color Images', flex_category: 'csp2_1'
 level 'v2 U2L4 Student Lesson Summary', progression: 'Lesson Vocabulary & Resources'
-named_level 'Images, Pixels, and RGB', progression: 'Images, Pixels, and RGB'
-level 'Pixelation - Lesson 15 - Complete 3-bit color', progression: 'Pixelation'
+level 'Images, Pixels, and RGB', progression: 'Images, Pixels, and RGB'
+level 'Pixelation - Lesson 15 - Complete 3-bit color', progression: 'Pixelation', named: true
```
Comment (dmcavoy) [inline dashboard/config/scripts/csp2-2017.script]: This got messed up - fix

File: dashboard/config/scripts/csp4-2019.script [hunk 3]
```diff
@@ -59,10 +59,10 @@ assessment 'U2L13 Free Response Wrap Up_2019', progression: 'Check Your Understa
...
 
 stage 'Encryption with Keys and Passwords', flex_category: 'csp4_1_2018'
-named_level 'CSP U4L06 SFLP_2019'
-named_level 'The Vigenere Cipher Widget_2019', progression: 'The Vigenere Cipher Widget'
-named_level 'How Secure Is Your Password?_2018_2019', progression: 'How Secure Is Your Password?'
-named_level 'Video - Encryption and Public Keys_2019', progression: 'Encryption and Public Keys'
+level 'CSP U4L06 SFLP_2019', named: true
+level 'The Vigenere Cipher Widget_2019', progression: 'The Vigenere Cipher Widget', named: true
+level 'How Secure Is Your Password?_2018_2019', progression: 'How Secure Is Your Password?', named: true
+level 'Video - Encryption and Public Keys_2019', progression: 'Encryption and Public Keys'
```
Comment (dmcavoy) [inline dashboard/config/scripts/csp4-2019.script]: this level is missing named

**Review by davidsbailey — APPROVED** [review]
Looks good! To make sure this is working, please turn off levelbuilder_mode in locals.yml and make sure that `cd dashboard ; rake seed:scripts` is passing locally, because drone only seeds a small subset of our scripts.

**Review by maddiedierker — APPROVED** [review]

**Comment by dmcavoy** [comment]
`rake seed:scripts` was successful. Going to merge this in.

**Pull request merged**